synapse/storage/prepare_database.py

```diff
@@ -28,7 +28,7 @@
 
 # Remember to update this number every time a change is made to database
 # schema files, so the users will be informed on server restarts.
-SCHEMA_VERSION = 56
+SCHEMA_VERSION = 57
```

Comment:
I don't think there's any need to bump the schema version?

Reply:
the comment implies I should?

Reply:
unfortunately I think the comment is wrong. If you read it at face value, we would bump it for every single delta file, which clearly we don't do.

The number should be updated when there is a backward-incompatible change, so that attempts to roll back will be met with a refusal to start rather than a subtle failure